sparkles/tests/test_review.py

```diff
@@ -507,3 +508,19 @@ def test_review_with_mon_star():
     assert aca.n_guide == 4
     assert len(aca.mons) == 1
     assert acar.messages == []
+
+
+def test_review_from_pickle():
+    """
+    Test that run_aca_review works from a gzip pickle file.
+    """
+
+    # This is similar to the loading from core.get_acas_from_pickle
+    # but that method does not allow full path specification.
+    filename = Path(__file__).parent / Path('data') / 'obsid.pkl.gz'
```

Comment:
The `Path()` around `data` is not needed (the `.parent` gives a Path object).

Reply:
Can you give the test file a more descriptive name such as `proseco_4.11.2_catalog.pkl.gz`?